src/shared/components/common/html-tags.tsx

```diff
@@ -2,7 +2,8 @@ import { htmlToText } from "html-to-text";
 import { Component } from "inferno";
 import { Helmet } from "inferno-helmet";
 import { httpExternalPath } from "../../env";
-import { getLanguages, md } from "../../utils";
+import { i18n } from "../../i18next";
+import { md } from "../../utils";
```

Comment:
This is newly imported, but I don't see it being used in your changes.

Reply:
`md` isn't newly imported, it was there before. `i18n` is newly imported, and it's used on line 24.

Reply:
Gotcha. The git changes made it look new.